client/app/dedicatedCloud/datacenter/drp/dedicatedCloud-datacenter-drp.service.js

```diff
@@ -0,0 +1,131 @@
+export default class {
+  /* @ngInject */
+  constructor($q, OvhApiDedicatedCloud, DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS) {
+    this.$q = $q;
+    this.OvhApiDedicatedCloud = OvhApiDedicatedCloud;
+    this.DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS = DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS;
+  }
+
+  getPccIpAddresses(serviceName) {
+    return this.OvhApiDedicatedCloud.Ip().v6().query({
+      serviceName,
+    }).$promise
+      .then(ipAddresses => ipAddresses.map(ipAddress => this.OvhApiDedicatedCloud.Ip().v6()
```

Comment:
Maybe I'm wrong but this is weird ? Shouldn't it be
```suggestion
      .then(ipAddresses => this.$q.all(ipAddresses.map(ipAddress => this.OvhApiDedicatedCloud.Ip().v6()
```
?